python/pyarrow/types.pxi

```diff
@@ -1774,6 +1774,43 @@ cdef class ExtensionType(BaseExtensionType):
         return ExtensionScalar
 
 
+cdef class JsonType(BaseExtensionType):
+    """
+    Concrete class for Arrow arrays of JSON data type.
```

Comment:
This is not the class for the array but for the type

Reply:
Changed to reflect that.